sdk/automation/Microsoft.Azure.Management.Automation/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Automation Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.")]
 
-[assembly: AssemblyVersion("3.0.4.0")]
-[assembly: AssemblyFileVersion("3.8.4.0")]
+[assembly: AssemblyVersion("3.0.5.0")]
```

Comment:
The new REST API contains breaking change, so please upgrade the version to 4.0.0

Reply:
updated the version to 4.0.0